mod6/assignment6.R

```diff
@@ -0,0 +1,25 @@
+A <- matrix(c(2, 0, 1, 3), ncol=2)
+B <- matrix(c(5, 2, 4, -1), ncol=2)
```

Comment:
While the `ncol` qualifier is necessary when skipping the usual second argument of `nrow`, if we generate a matrix given 4 observations and specify `nrow` as 2, it will still result in a 2x2 matrix.

Reply:
Resolved with c43910a, specifying `ncol=x` is equivalent to `nrow=x` for square matrices, which obviates the need for qualifying with the argument's name.